webpack.config.js

```diff
@@ -74,6 +74,25 @@ const webpackConfig = {
 				loader: 'babel-loader',
 				exclude: /node_modules/,
 			},
+			{
+				test: /\.js?$/,
+				use: {
+					loader: 'babel-loader',
+					options: {
+						presets: [
+							[ '@babel/preset-env', { loose: true, modules: 'commonjs' } ],
+						],
+						plugins: [ 'transform-es2015-template-literals' ],
+					},
+				},
+				include: new RegExp( '/node_modules\/(' +
+					'|d3-array' +
+					'|debug' +
+					'|regexpu-core' +
+					'|unicode-match-property-ecmascript' +
+					'|unicode-match-property-value-ecmascript)/'
```

Comment:
Does this list mean that every time we install a new package in the future we will have to check whether it works with IE11 and if not, add it here?

Reply:
Yes, unfortunately.  I'm not super happy about this either, but I'm not really sure if there are better ways.  The other options I looked at came with some pretty big downfalls.

I opened an issue [here](https://github.com/woocommerce/wc-admin/issues/1176) so we can discuss further and change this later if someone else knows a better way around this.